src/components/Notifications/NotificationsBehaviorGroupTable.tsx

```diff
@@ -207,7 +217,7 @@ export const NotificationsBehaviorGroupTable = ouia<NotificationsBehaviorGroupTa
                 </Tr>
             </Thead>
             <Tbody>
-                { rows }
+                { isEmpty() }
```

Comment:
The `isEmpty()` is kind of strange - I would preffer other name like `getTableContent()` or having the code inline here:

```tsx
   props.notifications.length === 0 ? <EmptyTableState /> : rows
```